src/lib/launchDarkly.js

```diff
@@ -1,9 +1,29 @@
 import launchDarklyBrowser from "ldclient-js";
+const url = require('url');
 
 export function ldBrowserInit (key, user) {
   return launchDarklyBrowser.initialize(key, user);
 }
 
+export function ldOverrideFlag(flagKey) {
+  let override;
+  /**
+   * Follow this overriding convention:
+   * @link https://git.corp.tc/capsela/tc_feature_flagging#overriding-feature-flags
+   **/
+  const query = url.parse(window.location.toString(), true).query;
+  const queryFlag = query["features." + flagKey];
```

Comment:
This works for supporting the advanced overriding, e.g., `?features.flagA=true` but I'm not seeing support for the basic list of boolean flags, e.g., `?features=flagA,flagB,flagC`.